### PR TITLE
Standardized naming in logs for errors.

### LINKED ISSUE #549: Standardize on the naming of the error object in logs
**What would you like to be added**:

Follow-up from #361, let's standardize on the way we do our naming in logs for errors.  In our code we do both of the following:

```js
// using `err`
logger.error({ err }, 'Some kind of error info...');

// using `error`
logger.error({ error }, 'Some kind of error info...');
```

Let's pick one (I prefer `err` but `error` is probably better) and do it everywhere.

**Why would you like this to be added**:

I was grepping through logs today, looking for errors, and it was made more difficult by the fact that we had two different names we use for reporting them.

### PR DESCRIPTION
**Pull request by LuigiZaccagnini**
## Issue This PR Addresses

Fixes #549

## Type of Change

- [X] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

This PR standardizes our naming in logs for **errors**. I renamed any **err** within the repository to **error**. I believe this is needed for the project because it helps everyone with finding errors, and it helps keep a standard style in the project. I decided that error would be a better name instead of err because it could help newer contributors understand the code better.   

## Checklist

- [X] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by humphd** [comment]
For example, see how this one is done: https://github.com/Seneca-CDOT/telescope/pull/565

File: src/backend/utils/wiki-feed-parser.js [hunk 1]
```diff
@@ -34,33 +31,29 @@ module.exports.parseData = function() {
...
+          };
+          objArray.push(obj);
+          feed = [];
+          obj = {};
         }
-      });
-      return objArray;
-    })
-    .catch(err => {
-      throw err;
```
Comment (c3ho) [inline src/backend/utils/wiki-feed-parser.js]: Do we still not need the ```.catch()``` block here since we are using ```.then()```?

Comment (humphd) [reply src/backend/utils/wiki-feed-parser.js]: No, if you leave it off, the caller will get the error.  If all you do is catch and re-throw, you've done the same thing as not having a catch at all.

Comment (c3ho) [reply src/backend/utils/wiki-feed-parser.js]: Whoops, for some reason I thought we were logging as well.

**Comment by humphd** [comment]
@c3ho would you be willing to talk with @LuigiZaccagnini on Slack about how to rebase this and fix those conflicts, since you know the code it affects pretty well?

**Comment by c3ho** [comment]
@humphd For sure, just ping me on slack and I'll answer as soon as I can.

**Comment by c3ho** [comment]
Think we got the same issue as before here. There's a lot of unrelated commits

**Comment by humphd** [comment]
The most likely cause of these other commits is that you're doing a `git pull`, which will merge, and you don't want that.  We can talk you through it in Slack.

**Comment by c3ho** [comment]
Alright I played around with his branch

This is what I did to undo his stuff
1. make sure master is up to date
1. git revert 11d2169f1341bc2818538044e6a35b13618c2653
2. git checkout issue-549
3. fix/accept the await changes in inactive-blog-filter.test.js
4. git rebase master -i

**Comment by c3ho** [comment]
Your PR from bfb0d6d was good and then it got weird from there. See if you can revert back to that one, rebase from master. It should be good from there. We can actually throw the problematic commits away by rebasing. 

1. make sure you're on branch issue-549 or whatever the working branch is. You can check this with ```git branch```
2. use ```git rebase master -i```
3. you should have something like this pop up:
pick Revert "Standardized naming in logs for errors." … 828e3c9

pick Revert "Standardized naming in logs for errors." … b0b596f

pick Revert "Standardized naming in logs for errors." … 41917ca

pick Revert "Standardized naming in logs for errors." … adc92b9

pick Revert "Standardized naming in logs for errors." … 96f2b56

pick Fixes #549 Standardized naming in logs for errors. bfb0d6d

pick Standardized naming in logs for errors. 28a1fad

pick Standardized naming in logs for errors. 7b018b1

pick Standardized naming in logs for errors. 29808eb

4. What you're going to do is replace the last three picks with drop so it looks like this:
pick Revert "Standardized naming in logs for errors." … 828e3c9

pick Revert "Standardized naming in logs for errors." … b0b596f

pick Revert "Standardized naming in logs for errors." … 41917ca

pick Revert "Standardized naming in logs for errors." … adc92b9

pick Revert "Standardized naming in logs for errors." … 96f2b56

pick Fixes #549 Standardized naming in logs for errors. bfb0d6d

drop Standardized naming in logs for errors. 28a1fad

drop Standardized naming in logs for errors. 7b018b1

drop Standardized naming in logs for errors. 29808eb

This will discard your last three commits which is where the issues start. This should get you back on track. I'd suggest you do this first, push the code and we'll take it from there.

The rules of rebasing
1. Always make sure you are on the master branch when you do ```git pull upstream master``` . You can check which branch you're on with ```git branch``` 
2. Switch to working branch with ```git checkout branchName``` in this case ```git checkout issue-549```
3. Rebase with ```git rebase master -i``` .

Git is hard, and it is frustrating, don't be discouraged. Dave was still helping students who are sometimes in their final year of studies with this stuff up until the end of the course with Git. After we sort this out your fix should almost be finished.

**Comment by c3ho** [comment]
Ok, we can work with this one. 
processor.js is good
queue.js is good
worker.js is good
**queue.js** is a maybe
redis.js is good
**shutdown.js** is not good
email-sender.js is good
**wiki-feed-parser.js** is not good
**inactive-blog-filter.test.js** is not good

So there are 4 files we'll need to fix.

1. make sure your master is up to date. ```git checkout master``` then ```git pull upstream master```
2. switch to working branch
3. rebase it before starting any work ```git rebase master -i``` When you do rebase make sure commits that don't belong there aren't in there. For example if you see some commit messages you never made, those probably aren't yours.
4. fetch a fresh copy of the four problem files
```git checkout master src/backend/utils/wiki-feed-parser.js```
```git checkout master test/inactive-blog-filter.test.js```
```git checkout master src/backend/lib/shutdown.js```
```git checkout master src/backend/lib/queue.js```
These should now all be up to date ones with no conflicts. Unfortunately we'll have to start from scratch on them
5. Apply your fixes to them, save the files, git add them, git commit them
6. push your working branch to origin and it should be solved.

Lastly, when we fix issues, only fix issues which are mentioned in the issue, and nothing more. I noticed in your very last commit, you changed the spacing in queue.js and email-sender.js... which have nothing to do with standardizing the errors. Only fix whatever is outlined in our issue and nothing more, in case changes have to be undone or a specific change starts to break stuff we can can pinpoint which fix did what, but if we start mixing other stuff that was fixed within an issue it gets confusing quickly.

**Pull request merged**